index.js

```diff
@@ -202,7 +202,8 @@ const rules = {
   "require-await": "error", // make it explicit
   "object-shorthand": [ "error", "properties" ], // make it short
   "object-curly-spacing": [ "error", "always" ], // consistency
-  "array-bracket-spacing": [ "error", "always" ] // consistency with above
+  "array-bracket-spacing": [ "error", "always" ], // consistency with above
+  "new-cap": 2 // consitency with test rules
```

Comment:
```suggestion
  "new-cap": "error" // consistency with test rules
```
`2` means error, right? Seems better to not mix option values for different rules.

Reply:
yes, it's error, I'll change